src/Support/StormpathLaravelServiceProvider.php

```diff
@@ -109,8 +109,8 @@ private function warmResources()
                 'href' => $mapping->accountStore->href,
                 'name' => $mapping->accountStore->name,
                 'provider' => [
-                    'href' => $mapping->accountStore->provider->href,
-                    'providerId' => $mapping->accountStore->provider->providerId,
+                  'href' => isset($mapping->accountStore->provider->href) ? $mapping->accountStore->provider->href : null,
+                  'providerId' => isset($mapping->accountStore->provider->providerId) ? $mapping->accountStore->provider->providerId : null,
```

Comment:
I'm a little concerned with this.  AFAIK, there should never be a case where these two values would not be set.